src/applications/vaos/new-appointment/components/ClinicChoicePage/index.jsx

```diff
@@ -77,6 +78,8 @@ export default function ClinicChoicePage({ changeCrumb }) {
     if (featureBreadcrumbUrlUpdate) {
       changeCrumb(pageTitle);
     }
+
+    dispatch(startDirectScheduleFlow({ isRecordEvent: false }));
```

Comment:
Just curious, why are we not recording this event?

Reply:
@simiadebowale - The though was I didn't want to record unnecessary events (duplicates) when the user navigates between workflows. The original events are still recorded. I will check with Peter to see if the event should be recorded each time the user switches between workflows.